recipes/rapidfuzz/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rapidfuzz" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: db0cd3b4cc024075362be92e247c0c2c7998bdbf9f775de4601717ba70b1bed9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```

Comment:
it looks like this package is py3 only. If so, add in the `build` section under `script`:

```
skip: True  # [py2k]
```

Reply:
thanks this is my first package and I am still a bit clueless :)